types/chrome-apps/index.d.ts

```diff
@@ -9597,6 +9602,8 @@ declare namespace chrome {
             isInternal: boolean;
             /** True if this display is enabled. */
             isEnabled: boolean;
+            /** True if the display is attached. */
+            activeState?: ActiveState
```

Comment:
@pyle  - Yes, these are in fact mandatory -  These are the changes that are to be submitted -https://chromium-review.googlesource.com/c/chromium/src/+/4644292/12/tools/typescript/definitions/system_display.d.ts#98. The existing docs for `DisplayUnitInfo` are [here](https://developer.chrome.com/docs/extensions/reference/system_display/#type-DisplayUnitInfo) - activeState will be added shortly.

Is there harm in making this a mandatory property, even before all builds have the property available?

Reply:
There _shouldn't_ be any harm unless they attempt to access the property and it doesn't exist in the chrome binary yet (the same would happen if it was optional and they didn't check for existence). 


Personally, i would wait for submission of the docs/implementation into main branch before submitting the typing to avoid any confusion; but the likelihood of someone discovering and attempting to use this API is low and they'd get an error during development so it's probably fine.